omnibus/config/software/postgresql12.rb

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright:: Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name "postgresql12"
+
+default_version "12.5"
+
+license "PostgreSQL"
+license_file "COPYRIGHT"
+skip_transitive_dependency_licensing true
+
+dependency "zlib"
+dependency "openssl"
+dependency "libedit"
+dependency "ncurses"
+dependency "libossp-uuid"
+dependency "config_guess"
+
+source url: "https://ftp.postgresql.org/pub/source/v#{version}/postgresql-#{version}.tar.bz2"
+version("12.5") { source sha256: "bd0d25341d9578b5473c9506300022de26370879581f5fddd243a886ce79ff95" }
+version("9.6.15") { source sha256: "3cd9fe9af247167f863030842c1a57f58bdf3e5d50a94997d34a802b6032170a" }
```

Comment:
Might not be required here?

Reply:
yeah if this is 12 we should actually just nuke this bit

Reply:
Nuked.